service/asset_service.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"context"
+
+	"github.com/vkhichar/asset-management/repository"
+)
+
+type AssetService interface {
+	CreateAsset(ctx context.Context, assetCategory string, assetStatus string, purchaseCost float64, assetName string, spec *domain.Specs) (*domain.Asset, error)
+}
+
+type assetService struct {
```

Comment:
@vkhichar We can name implementing struct as AssetServiceImpl right?

Reply:
no need to add `Impl` suffix, It's understood that it's an implementation, without putting the suffix too
though, there is no standard for this naming convention